app/models/taxon_lineage.rb

```diff
@@ -34,6 +34,49 @@ class TaxonLineage < ApplicationRecord
   PHAGE_FAMILIES_TAXIDS = [10_472, 10_474, 10_477, 10_656, 10_659, 10_662, 10_699, 10_744, 10_841, 10_860,
                            10_877, 11_989, 157_897, 292_638, 324_686, 423_358, 573_053, 1_232_737].freeze
 
+  # From https://www.niaid.nih.gov/research/emerging-infectious-diseases-pathogens
+  # Accessed 9/18/2018.
+  # Categories are listed in order of priority (A, B, C). So if a taxon is included by the rules of both
```

Comment:
We should eventually put this in the database, probably as metadata.

Reply:
sure